files/general/etc/nginx/nginx.conf

```diff
@@ -33,7 +33,7 @@ http {
         index index.php index.html;
 
         location / {
-            try_files $uri $uri/ /index.php;
+            try_files $uri $uri/ /index.php?$args;
```

Comment:
Nice catch! Could you please use `$is_args` instead of `?` so we do not send a `?` all the time?